openpype/settings/entities/schemas/projects_schema/schemas/schema_anatomy_imageio.json

```diff
@@ -386,6 +386,13 @@
                             "type": "text",
                             "key": "viewTransform",
                             "label": "Viewer Transform"
+                        },
+                        {
+                            "type": "path",
+                            "key": "policyFilePath",
+                            "label": "Maya color managment policy file path",
+                            "multiplatform": true,
+                            "multipath": true
```

Comment:
Multipath can be false in case there is no necessity for multiple path to be set.

Reply:
which I feel is also case of `configFilePath` there is no reason for multiple path inputs if only one is used.